doc/kubernetes/modules/ROOT/pages/running/metrics/keycloak_service_level_indicators.adoc

```diff
@@ -0,0 +1,142 @@
+= {project_name} Service Level Indicators
+:description: This document contains details of the SLI's to monitor your {project_name} deployment's performance.
+
+Service Level Objectives (SLOs) and Service Level Indicators (SLIs) are essential components in monitoring and maintaining the performance and reliability of {project_name} in production environments.
+
+- **SLOs** define the target performance or reliability level that {project_name} should achieve, such as availability or response times.
+- **SLIs** are the specific metrics used to measure how well {project_name} is meeting these targets, such as uptime percentage or request latency.
+
+By tracking these, {project_name}
+users can ensure that their deployments are running optimally and detect potential issues early.
+These metrics are also invaluable for teams supporting {project_name},
+enabling effective troubleshooting and performance tuning.
+
+We assume that the scenario
+for defining the SLOs and SLIs is based on the below steps and also the histogram metrics are enabled in {project_name} for the PromQL queries
+to run as intended.
+
+====
+As a {project_name} user,
+
+* I want to be able to log in,
+* refresh my token,
+* and log out.
+
+So that I can interact with the {project_name} system effectively and perform the necessary tasks without interruption.
+====
+
+
+[cols="2,1,2,2", options="header"]
+|===
+| Service Level Objective^*^ | Service Level Indicator | SLI Metric         | Metric Details
+
+| {project_name} should be available 99.99% of the time.                 | Availability            | Uptime percentage is the ratio of successful authentication requests to total authentication requests.               |
+The `up` metric which indicates if the Prometheus server is able to scrape metrics from the {project_name} instance.
+This metric will have a value of 1 if the {project_name} service is available
+and responding to Prometheus scrape requests,
+and 0 if the service is down or unreachable.
+
+It could be represented in a pseudo PromQL query as `sum(up{...}) > 0`
+
+| 95% of {project_name} authentication requests should have a latency below 200ms. | Authentication Latency  | {project_name} server-side metrics to track latency for specific endpoints along with Response Time Distribution.            | `http_server_requests_seconds_bucket`, `http_server_requests_seconds_count`.
```

Comment:
there is no 200ms bucket in the metrics, should we change this to 250ms?

Reply:
I'm currently reworking this, and will update it 250 ms

Reply:
ack, the doc looks good to me.